### ISSUE TITLE
Reseller information editable for clients

### ISSUE THREAD

**Issue by SjamonDaal** [openprovider/OP-WHMCS7#11]
By default, customers are able to change the reseller information.
This should never be possible from the client panel, it should be optional in the admin panel.

Please fix this.

Temp Fix:
Edit: site/modules/registrars/openprovider/openprovider.php
Go to line: 777
Add the following line on line 778: `unset($values["Reseller"]);`

Preview:
```
function openprovider_GetContactDetails($params)
{
    try
    {
     	$domain             =   new \OpenProvider\API\Domain(array(
            'name'          =>  array_key_exists('original',$params) ? $params['original']['sld'] : $params['sld'],
            'extension'     =>  $params['tld']
        ));

        $api                =   new \OpenProvider\API\API($params);
        $values             =   $api->getContactDetails($domain);
        unset($values["Reseller"]);
    }
    catch (\Exception $e)
    {
        $values["error"] = $e->getMessage();
    }

    return $values;
}
```

**Comment by MrKoopie** [comment]
With which tld do you have this issue?

**Comment by SjamonDaal** [comment]
I've tested this with .nl .be and .com

**Comment by soberoctober** [comment]
fixed in [v3.0](https://github.com/openprovider/OP-WHMCS7/tree/v3.0)
